### ISSUE TITLE
Possibility to run this outside of eclipse

### ISSUE THREAD

**Issue by infinity0** [zhourenjian/java2script#5]
Hello Renjian, I'm trying to package Jmol for Debian, but it seems that they require the user to run java2script as an eclipse plugin. This is very hard to automate in a script, so we can't do this. I'm wondering what sort of work would be needed to run java2script outside of eclipse, as a standalone program?

**Comment by infinity0** [comment]
In case other people are wondering: this already exists, here: https://github.com/zhourenjian/java2script/blob/master/incubator/net.sf.j2s.ui.cmdline/docs/j2s-cmdline-api/j2s-cmdline-api.xml

I haven't tested it out yet but from reading the docs, it's supposed to work in quite a lot of different use-cases.

I've already created some Makefiles to automatically build j2s, including the cmdlineApi, outside of eclipse, here:

https://github.com/infinity0/java2script/blob/j2s-eclipse-3.8/Makefile

The built plugin jars will work against Eclipse 3.8. I chose 3.8 because that is what is available in Debian, but someone could probably re-do my approach with the latest version of Eclipse.
